clang/lib/StaticAnalyzer/Core/BugSuppression.cpp

```diff
@@ -119,6 +121,28 @@ class CacheInitializer : public DynamicRecursiveASTVisitor {
   Ranges &Result;
 };
 
+std::string timeScopeName(const Decl *DeclWithIssue) {
+  if (!llvm::timeTraceProfilerEnabled())
+    return "";
+  return llvm::formatv(
+             "BugSuppression::isSuppressed init suppressions cache for {0}",
+             DeclWithIssue->getDeclKindName())
+      .str();
+}
+
+llvm::TimeTraceMetadata getDeclTimeTraceMetadata(const Decl *DeclWithIssue) {
+  assert(DeclWithIssue);
+  assert(llvm::timeTraceProfilerEnabled());
+  std::string name = "<noname>";
```

Comment:
Nit: start variables with uppercase in this function and some others below.

Reply:
fixed
b5792c2b135e [NFC] Uppercase variable names